.github/workflows/verify-browserstack.yml

```diff
@@ -5,7 +5,13 @@ on: pull_request
 jobs:
   verify-browserstack:
     name: Verify Browserstack
+
+    # Run the job if it's not triggered by a PR or if the PR is from the same repository
+    # This is because this job requires a secret, and secrets cannot be shared with forks
+    if: github.event.pull_request == null || github.event.pull_request.head.repo.full_name == github.repository
```

Comment:
Sorry to revive this after so long, but...
On what occasion does `github.event.pull_request == null` occur?
We just have `on: pull_request` as trigger.

Reply:
Hi @abdonrd,

I don't know. I actually [copied this from lit](https://github.com/lit/lit/blob/4395e3d45d68cf704155b507b2e0607fc7759a0c/.github/workflows/benchmarks.yml#L10-L13) on your suggestion in https://github.com/modernweb-dev/web/pull/824#issuecomment-723416910, and lit uses `on: [pull_request]` also. Probably it would be fine to just use:

```yaml
    if: github.event.pull_request.head.repo.full_name == github.repository
```